cdap-app-fabric/src/main/java/io/cdap/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java

```diff
@@ -694,6 +695,31 @@ private void doGetSchedule(HttpResponder responder, String namespace,
     responder.sendJson(HttpResponseStatus.OK, GSON.toJson(detail, ScheduleDetail.class));
   }
 
+  /**
+   * Get the metadata of the schedule of an application.
+   */
+  @GET
+  @Path("apps/{app-name}/schedules/{schedule-name}/metadata")
```

Comment:
can we just add the new information to the ScheduleDetail object instead of adding separate endpoints?

Reply:
I think we can. But went with separate objects approach just to be consistent with current style of having ProgramSchedule and ProgramScheduleMeta.